cmd/ocsp-updater/main.go

```diff
@@ -111,7 +111,7 @@ func newUpdater(
 	stalenessHistogram := prometheus.NewHistogram(prometheus.HistogramOpts{
 		Name:    "ocsp_status_staleness",
 		Help:    "How long past the refresh time a status is when we try to refresh it. Will always be > 0, but must stay well below 12 hours.",
-		Buckets: []float64{10, 100, 1000, 10000, 43200},
+		Buckets: []float64{10, 100, 1000, 10000, 21600, 32400, 36000, 39600, 43200},
```

Comment:
Feels slightly odd to have some buckets are are powers of ten seconds, and some buckets that are multiples of hours. But since 10k seconds is _pretty_ close to 3 hours, not quite odd enough to turn this bus around :P